src/Microsoft.DotNet.Darc/src/DarcLib/GitSubmoduleInfo.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+namespace Microsoft.DotNet.DarcLib;
```

Comment:
Please be consistent with the style of this repository when making changes. None of the other files use the namespace declaration statements.

Reply:
In my opinion this is one of the harmless new features that we can start adopting for new files easily?
Don't get me wrong, I'd never go against variable naming and follow whatever but file-scope namespaces are just so much good when doing code reviews and you have less indentation..

I can switch all these new files and we can switch the whole repo to file-scoped first but I would strongly advocate to use this new style though.

Reply:
I'm fine if we want to switch to it in the repo. It would take only a few minutes, since it's a purely mechanical change, but style varying from file to file just adds difference for no real benefit.  Style is only really useful if it's consistent.